src/installation.rst

```diff
@@ -15,15 +15,22 @@ Cylc runs on Unix-like systems including Linux and Mac OS.
 Via Conda (recommended)
 ^^^^^^^^^^^^^^^^^^^^^^^
 
+.. warning::
+
+   The ``conda`` environment solver can be very slow and memory-intensive for
+   complex packages. We recommend installing the much faster ``mamba`` solver
+   first, and then installing Cylc with that.
+
 .. code-block:: sub
 
-   $ conda install -c conda-forge cylc-flow
+   $ conda install -c conda-forge mamba
```

Comment:
This suggests installing `mamba` into the cylc env (or cylc into the base env).

Reply:
I see I have an extra erroneous line in there (fixed) ... but it is OK to install mamba into the cylc env.

Reply:
The docs says conda and mamba will only work when installed in the "base" env. You can install them into other envs but I guess they won't work for environment management, dunno.